regression-test/suites/nereids_syntax_p0/transform_outer_join_to_anti.groovy

```diff
@@ -62,5 +62,25 @@ suite("transform_outer_join_to_anti") {
         sql("select eliminate_outer_join_B.* from eliminate_outer_join_A right outer join eliminate_outer_join_B on eliminate_outer_join_B.b = eliminate_outer_join_A.a where eliminate_outer_join_A.null_a is null")
         contains "OUTER JOIN"
     }
+
+    explain {
```

Comment:
does this will do or-expansion and side b can be converted to anti?

Reply:
`orExpansion` is behind `convertToAnti`, so it is not supported right now.